CHANGELOG.md

```diff
@@ -1,5 +1,8 @@
 # mediatimestamp Changelog
 
+## 1.6.0
+- Removed the PRESERVE_START and PRESERVE_END rounding options which are used in TimeRange.normalise().
```

Comment:
Does this make a breaking change (to anyone relying on PRESERVE_START/PRESERVE_END and trigger a major version bump? Or are we happy it's not used anywhere else (https://github.com/search?q=TimeRange.PRESERVE_START&type=Code suggests it isn't)

Reply:
I thought about that but went for a minor change only because it is only used in one place and there is a job open to move to mediatimestamp v2.0.0-dev branch and so I didn't want to trigger that change now.